object_real: false
iiwa_real: false
manual_mode: false
hollow: false
debug: true
inertia: true


#average of desired workspace of iiwa1 wrt world frame. In real, world frame is exactly between iiwa_base so centers should have x =/= 0
# SIM CENTERED
center1: [0.55, -0.4, 0.185]  #z is center of box
center2: [0.55, 0.4, 0.185]
# SIM TURNED AXIS
# center1: [-0.05, 0.95, 0.185]  #z is center of box
# center2: [0.74, 0.95, 0.185]
# REAL
# center1: [-0.95, 0.05, 1.07]  #z is center of box
# center2: [-0.95, 0.70, 1.07]
    
ee_offset:
  h: 0.2                # horizontal offset for the end effector behind the object's center before moving to hit
  v: 0.025              # vertical offset

hittable:               # to create a hittable area in the form of an ellipsoid with its center at an offset from center1 (or center2)
  reach:
    x: 0.12             # perp to centerline     
    y: 0.25             # along centerline
  offset: 
    x: 0.0              # currently not useful as it flips x direction for other robot
    y: 0.1              # positive is inwards along centerline

table:
  properties:           # keep in mind that the iiwa base has a max radius of 0.3
    size:
      # x: 2.4
      # y: 0.8
      x: 0.8
      y: 2.4
      z: 0.05
    COM:
      x: 0.0
      y: 0.0
      z: 0.0
    mass: 100
    mu: 0.5
    mu2: 0.5
  initial_pos:
    # x: 0.55
    # y: 0.89
    # z: 0.025
    x: 0.55
    y: 0.0
    z: 0.025

box:
  properties:
    size:
      x: 0.26
      y: 0.26
      z: 0.23
    COM:            #relative to geometric center
      x: 0.0
      y: 0.0
      z: 0.0
    mass: 0.363
    mu: 0.075
    mu2: 0.075
  initial_pos:
    # x: 0.65
    # y: 0.95
    # z: 0.175
    x: 0.55
    y: -0.4
    z: 0.175
  
mini:
  properties:
    size:
      x: 0.1
      y: 0.1
      z: 0.1
    COM:            #relative to geometric center
      x: 0.0
      y: 0.0
      z: 0.0
    mass: 0.3
    mu: 0.03
    mu2: 0.03
  initial_pos:      #relative to big box center
    x: 0.0
    y: -0.065
    z: -0.065



hit:
  speed: 0.6
  direction: 0.0
